nncf/torch/quantization/layers.py

```diff
@@ -997,6 +997,8 @@ def get_quantizer_config(self) -> QuantizerConfig:
 
 
 def get_per_channel_scale_shape(input_shape, is_weights, channel_idx: int = None):
+    # TODO: case channel_ids=0, is_weights=True and per_sample_stats=True
+    # leads to dimension error in statistic calculation
```

Comment:
Why not solve it?

Reply:
I was wrong: 
"""
    Assumes that input_shape is supplied in either [B, C, H, W] or [N_out, N_in, H, W] format,
    or derivatives.
"""
so case  channel_idx=0, is_weights=False and per_sample_stats=True (I made a mistake in comment) is impossible